packages/store/src/utils/utils.ts

```diff
@@ -64,3 +64,16 @@ export const setValue = (obj: any, prop: string, val: any) => {
  */
 export const getValue = (obj: any, prop: string): any =>
   prop.split('.').reduce((acc: any, part: string) => acc && acc[part], obj);
+
+/**
+ * Boolean conversion:
+ *
+ *    like '', 'false' to boolean
+ *    ngxsFormClearOnDestroy, ngxsFormClearOnDestroy="true" should be treated as true
+ *    ngxsFormClearOnDestroy='false' should be treated as false
+ *
+ * @ignore
+ */
+export function coerceBoolean(val: any) {
```

Comment:
Add unit test with `ngxsFormClearOnDestroy='false'` and `ngxsFormClearOnDestroy='true'`

Reply:
This also can be moved out of `store` and stored directly in the form plugin's code.

Reply:
Agreed, this utility function should not be added to or exposed from the core lib, but rather added as a private function within the form plugin.

Reply:
should I create utils file inside form plugin and also sperate file for utils.spec.ts

Reply:
Please just put this code inline in the `set clearDestroy(val: boolean) { ...` function.
There is no need to extract a utility with just a single usage.